tutorials/examples/train_hypergrid_simple.py

```diff
@@ -49,16 +47,13 @@
 optimizer = torch.optim.Adam(gflownet.pf_pb_parameters(), lr=1e-3)
 optimizer.add_param_group({"params": gflownet.logz_parameters(), "lr": 1e-1})
 
-n_iterations = int(1e4)
-batch_size = int(1e5)
-
-for i in (pbar := tqdm(range(n_iterations))):
+for i in (pbar := tqdm(range(1000))):
     trajectories = sampler.sample_trajectories(
         env,
-        n_trajectories=batch_size,
+        n_trajectories=16,
         save_logprobs=False,
         save_estimator_outputs=True,
-        epsilon=exploration_rate,
```

Comment:
Why did you remove this (and `batch_size` and `n_iterations`)?

Reply:
Sorry, I didn't mean to include this change initially—it was unintentionally committed with the whole file. However, after taking another look, I think there’s room for improvement, like using argparse instead of relying on global variables. I’ll push this update in the next commits!

Reply:
Totally agree with that!!